tests/services/components/test_metadata_component.py

```diff
@@ -10,7 +10,7 @@
 from invenio_rdm_records.proxies import current_rdm_records
 from invenio_rdm_records.records import RDMRecord
 from invenio_rdm_records.records.api import RDMDraft
-from invenio_rdm_records.services.components import MetadataComponent
+from invenio_rdm_records.services.components.metadata import MetadataComponent
```

Comment:
same, is a matter of preference... but also helps to keep import statements short 👼